Wikipedia/Code/SavedViewController.swift

```diff
@@ -310,6 +310,14 @@ extension SavedViewController: CollectionViewEditControllerNavigationDelegate {
         let editingStates: [EditingState] = [.swiping, .open, .editing]
         let isEditing = editingStates.contains(newEditingState)
         actionButton.isEnabled = !isEditing
+        if (newEditingState == .open),
+            let batchEditToolbar = self.savedArticlesViewController?.editController.batchEditToolbarView,
+            let contentView = self.containerView,
+            let appTabBar = self.parent?.view.subviews.first(where: {($0 as? UITabBar) != nil}) {
```

Comment:
For this one, I would rather reference `appTabBar` a bit more strongly. Maybe via a delegate that has a `tabBar` property that `AppViewController` conforms to. Reason being should we ever present `SavedViewController` in a different context, say without a parent that has a tab bar, this would fail silently. Whereas if `SavedViewController` must be instantiated with a delegate that has a tab bar, the developer would know right away that something is up, i.e. this VC has an assumption that it can point to a tab bar.

Reply:
That sounds much better to me as well. 

As I alluded to in the PR, do you think adding a new delegate to the codebase and setting it here is worth the minor improvement in VoiceOver support? This PR allows the edit view of the saved article VC to be read in a more logical order by VoiceOver (when swiping through items). If we can't do it easily, should we just close it out and leave it as is? I'm not sure how much code this minor improvement is worth. My inclination is that if we can't do it easily, it's not worth doing. But I'm very open to other thoughts around importance.

Reply:
I say forge ahead, I think a slight VoiceOver improvement is always better than nothing and I don't think the code changes will be too heavy at this point. If it feels like it's sending you down a big rabbit hole we can talk about maybe punting until after the audit since that'll probably lead to additional changes anyways.

Reply:
Sounds good. I updated this. I put the protocol in a new file - let me know if we should tuck it elsewhere instead. Thanks!